learn/educators/nodejs.md

```diff
@@ -0,0 +1,127 @@
+
+Read below for recommendations for extensions, settings, and a link to a free lesson that you can use in your classes.
+
+## Intro to Web Development at Naresuan University
+
+[Charles Allen](https://th.linkedin.com/in/ajahncharles) is a lecturer at [Naresuan University](https://www.nu.ac.th/) in Thailand, where he teaches several Computer Science and Business courses. He teaches a project-based Web Development course where students build a TypeScript app using the Vue.js framework and Firebase.
+
+### Why Visual Studio Code?
+
+Charles has been using Visual Studio Code to teach for the past 4 years since he inherited the class. In his own words:
+
+> "The more realistic the learning environment, the more comprehensive the education. I am not just teaching code; I'm also teaching the tools, tactics, and teamwork my students will need to start building for themselves. VS Code is the first choice of web professionals, so it’s the first choice for my class!"
+
+In 2019, [StackOverFlow found VS Code to be the most popular editor](https://insights.stackoverflow.com/survey/2019#development-environments-and-tools) amongst their community.
+
+### Portable setup
+
+VS Code is great for Charles because it has a [portable mode](/docs/editor/portable), which stores all of its app data in one spot. He prepares a single “image”, [available on GitHub](https://github.com/AjahnCharles/lab-image-win), containing his favorite extensions, settings, and cached versions of some libraries. This can be quickly copied to every lab machine without needing to run installers, download additional files, or manually configure each machine.
+
+### Classroom settings
+
+The extensions he likes to have installed:
+* [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) - his favorite! auto-formatting keeps students focused on program logic and linting helps them quickly find, understand, and fix errors in their JavaScript code
+* [TabNine](https://marketplace.visualstudio.com/items?itemName=TabNine.tabnine-vscode) - longer AI-driven code-snippets helps non-native English speakers avoid code-breaking typos and reduces the impact of slow typing
+* [REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) - share API designs as .http files and simplify manual API testing within VS Code
+* [Vetur](https://marketplace.visualstudio.com/items?itemName=octref.vetur) - Vue.js framework support for syntax highlighting, code completion, and more
+
+These extensions are also part of the Node.js and JavaScript Education Extension Pack below!
+
+
+And here are his settings, which you can use by editing the [settings file](/docs/getstarted/settings.md) and pasting these in:
+
+```json
+{
+    // Formatting & Linting
```

Comment:
This comment heading seems wrong.
I also don't see this setting in his settings list? Do we add this because we want to promote the untitled file option for an empty editor?

Reply:
I'm not sure what you mean by:
> Do we add this because we want to promote the untitled file option for an empty editor?

Like asking should we promote new file on startup of VS Code?

Reply:
I looked through his GitHub repo until I found https://github.com/AjahnCharles/lab-image-win/blob/main/devtools/vscode/data/user-data/User/settings.json. 
I didn't see `workbench.startupEditor` listed so I thought you might have added this setting.

Reply:
Ohh this is what he had originally provided me, so I copied the version he sent me personally. I can change it to this version on his repo :)